.github/workflows/rebuildServers.yml

```diff
@@ -0,0 +1,66 @@
+name: Re-build and re-deploy on test & production servers && trigger CCDH model refresh
+
+on:
+  workflow_dispatch:
+
+jobs:
+  rebuild_redeploy:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Run script on server
+      # shell: bash  # Usage: this ensures that env variable usage syntax is correct; Disabled because: causes yaml file syntax error
+      uses: appleboy/ssh-action@master  # https://github.com/appleboy/ssh-action
+      with:
+        host: terminology.ccdh.io
+        username: docker
+        password: ${{ secrets.DOCKER_USER_PASSWORD }}
+        # key: ${{ secrets.KEY }}  # not needed
+        # port: ${{ secrets.PORT }}  # default: 22
+        # TODO: Is 'exit' necessary? I'm not 100% sure, but the last time I ran
+        #   the action, it seemed to stay running indefinitely. - jef 2021/07/15
+        # timeout: 30s  # timeout - timeout for ssh to remote host, default is 30s
+        command_timeout: 60m  # command_timeout - timeout for ssh command, default is 10m
+        # TODO: Since we run this in test and production, should we create a
+        #   .sh script for this and re-use the code instead of have it copied
+        #   2x here? - jef 2021/07/26
+        script: |
+          # 1. Test env
+          # - Update codebase
+          cd /opt/Projects/ccdh-terminology-service-test  # test env
+          # git checkout master
+          git pull
+          # - Deploy
+          cd docker
+          docker-compose down
+          docker-compose build ccdh-api
+          docker-compose up
+          docker exec -it ccdh-api /bin/bash
+          python -m ccdh.importers.importer
+          # python -m ccdh.integrations.crdc_h  # only do this on prod
+
+          # TODO: Wait until this is done, and trigger build production
+          # Options:
```

Comment:
@jiaola Actually I want to ask you your opinion on how I should implement "(1) build test server, and then, on success, (2) build production server".

I'm thinking about the best implementation for making sure that steps 1 and 2 go off cleanly and in order as described. I wrote some comments inside `rebuildServers.yml` with my thoughts. I'm thinking of going for approach (a)(B), just because I think that will be the easiest for me, and it also leaves us with the benefits of: (i) composable, modular GH actions, (ii) a useful `ENVIRONMENT_NAME` variable.

Reply:
I've merged this already, but interested on your thoughts, if you have any. If you think I should just go with what I think is best, I'm fine with that, too!